vspace/src/main/webapp/WEB-INF/views/staff/exhibit/config.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ page pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="t"%>
+<%@ taglib prefix="sec"
+	uri="http://www.springframework.org/security/tags"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+<h1>Exhibition Configuration</h1>
+
+<div style="padding-bottom: 20px;">
+	<c:url value="/staff/exhibit/config" var="postUrl" />
+	<c:if test="${param.success eq true}">
+    	<div class="alert alert-success">
+  			<strong>Successfully saved!</strong>
+		</div>
+	</c:if>
+	<c:if test="${param.success eq false}">
+    	<div class="alert alert-danger">
+  			<strong>Error: Could not save default space!</strong>
+		</div>
+	</c:if>
```

Comment:
instead of putting an alert like this into every page that needs it, let's just put it in the skeleton. type and message should be taken from the model: if there is a property (like showAlert) set in the model, show alert box and use a property from the model that defines what type it is (e.g. danger, info, warning, etc) and one property that holds the message.